py/dml/expr.py

```diff
@@ -190,6 +191,20 @@ def writable(self):
 
 mkLit = Lit
 
+class NullExpr(Expression):
```

Comment:
Naming: other expression classes don't end with `Expr`. I suggest `Null` or `NullConstant` or `NullPointerConstant`

Reply:
`NullConstant` it is.